apps/src/util/curriculumRecommender/curriculumRecommender.js

```diff
@@ -0,0 +1,173 @@
+import {
+  IMPORTANT_TOPICS,
+  UTC_PUBLISHED_DATE_FORMAT,
+  FAKE_RECOMMENDER_SCORING,
+} from './curriculumRecommenderConstants';
+import moment from 'moment';
+
+const now = moment().utc();
+const oneYearAgo = moment()
+  .utc()
+  .year(now.year() - 1);
+const twoYearsAgo = moment()
+  .utc()
+  .year(now.year() - 2);
+
+/*
+ * Curriculum recommenders: Each recommender receives an array of all curricula to consider and any preferences it should prioritize. The given
+ * recommender will score each curriculum based on the questions associated with that recommender. If a curriculum satisfies the conditions of a
+ * question, it receives points based on that recommender's scoring framework found in 'curriculumRecommenderConstants.js'. Once all curricula
+ * have been scored, the recommender returns an array of the passed-in curricula sorted in descending order by score (ties are broken by
+ * prioritizing featured curricula and more recently published curricula).
+ */
+export const getTestRecommendations = (
+  curricula,
+  duration,
+  marketingInitiative,
+  schoolSubjects,
+  csTopics
+) => {
+  const curriculaScores = [];
+  curricula.forEach(curriculum => {
+    let score = 0;
+    score += hasDesiredDuration(FAKE_RECOMMENDER_SCORING, curriculum, duration);
+    score += hasDesiredMarketingInitiative(
+      FAKE_RECOMMENDER_SCORING,
+      curriculum,
+      marketingInitiative
+    );
+    score += hasAnySchoolSubject(FAKE_RECOMMENDER_SCORING, curriculum);
+    score += hasDesiredSchoolSubjects(
+      FAKE_RECOMMENDER_SCORING,
+      curriculum,
+      schoolSubjects
+    );
+    score += hasImportantButNotDesiredTopic(
+      FAKE_RECOMMENDER_SCORING,
+      curriculum,
+      csTopics
+    );
+    score += hasDesiredTopics(FAKE_RECOMMENDER_SCORING, curriculum, csTopics);
+    score += howRecentlyPublished(FAKE_RECOMMENDER_SCORING, curriculum);
+    curriculaScores.push([curriculum, score]);
+  });
+  console.log('SEPARATOR');
```

Comment:
Some console logs here:)

Reply:
Whoops! I removed them in a branch I started off this one but not the original, thank you!